modules/images-update-global-pull-secret.adoc

```diff
@@ -25,18 +25,43 @@ As of {product-title} 4.7, changes to the global pull secret no longer trigger a
 
 .Prerequisites
 
-* You have a new or modified pull secret file to upload.
 * You have access to the cluster as a user with the `cluster-admin` role.
 
+
 .Procedure
+To update the global pull secret, complete the following steps:
+[NOTE]
+====
+To overwrite your existing pull secret using a new pull secret file, skip steps 1 & 2.
+====
+
+. Download the pull secret:
++
+[source,terminal]
+----
+$ oc get secret/pull-secret -n openshift-config --template='{{index .data ".dockerconfigjson" | base64decode}}' ><pull_secret_location> <1>
+----
+<1> Provide the path to the pull secret.
+
+. Add the new pull secret:
++
+[source,terminal]
+----
+$ oc registry login --registry="<registry>" \ <1>
+--auth-basic="<username>:<password>" \ <2>
+--to=<pull-secret-location>
+----
+<1> Provide the new registry.
+<2> Provide the credentials of your new registry.
++
+Alternatively, you can perform a manual update to the pull secret file.
 
-* Enter the following command to update the global pull secret for your cluster:
+. Update the global pull secret for your cluster:
 +
 [source,terminal]
 ----
-$ oc set data secret/pull-secret -n openshift-config --from-file=.dockerconfigjson=<pull-secret-location> <1>
+$ oc set data secret/pull-secret -n openshift-config --from-file=.dockerconfigjson=<pull_secret_location>
 ----
-<1> Provide the path to the new pull secret file.
```

Comment:
Why are you removing the annotation for the variable? Unless there's a strong reason to do so, please restore this information.

Reply:
L52 has the same placeholder id hence removed since it has already been introduced in the previous step

Reply:
@kalexand-rh L52 has the same placeholder id hence removed since it has already been introduced in the previous step. Let me know if I need to re-annotate.

Reply:
You do need to re-annotate because not every user is going to complete the command at line 52.